webpack/assets/javascripts/react_app/components/notifications/Notification.js

```diff
@@ -18,6 +18,33 @@ const Notification = ({created_at, seen, text, level, id}) => {
     (<strong {...tooltip} className="drawer-pf-notification-message pointer"
              onClick={markAsRead}>{text}</strong>);
 
+  const links = [];
+
+  let actions_dropdown = '';
+
+  if (actions.links != null) {
+    for (let i = 0; i < actions.links.length; i++) {
```

Comment:
i think we can easily implement this component ourself using a stateful component, more reason for it to be on it's own